pkg/hub/submarinerbrokerinfo/brokerinfo.go

```diff
@@ -31,6 +31,7 @@ const (
 	catalogName                   = "submariner"
 	defaultCatalogSource          = "redhat-operators"
 	defaultCatalogSourceNamespace = "openshift-marketplace"
+	DefaultCatalogChannel         = "stable-0.15"
```

Comment:
This should be private (see my comment on the unit test):

```suggestion
	defaultCatalogChannel         = "stable-0.15"
```